release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -409,6 +409,29 @@ Be aware that CIDR ranges for networks are not adjustable after cluster installa
 
 If you are using the OVN-Kubernetes cluster network provider, you can now enable IPsec encryption after cluster installation. For more information about how to enable IPsec, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-11-new-metallb-CRDs-logging-control"]
+==== Support for new MetalLB custom resource defintions (CRDs) and control over logging verbosity
```

Comment:
You could shorten this heading by changing "custom resource defintions (CRDs)" to "CRDs". Then, you could provide the full name of the acronym when you first use it in the main text in the section.